frontend/src/utils/MadLibs.ts

```diff
@@ -19,6 +19,8 @@ import { SHOW_PHRMA_MENTAL_HEALTH } from '../data/providers/PhrmaProvider'
 import { SHOW_GUN_VIOLENCE } from '../data/providers/GunViolenceProvider'
 import { GEORGIA_FIPS, USA_FIPS } from '../data/utils/ConstantsGeography'
 import { FIPS_MAP } from '../data/utils/FipsData'
+import { title } from 'vega-lite/build/src/channeldef'
```

Comment:
remove this?

Reply:
@JDemlow I think this import is a mistake right?

Reply:
I don't recognize it. Maybe VSCode tried an auto import? I don't remember explicitly adding that. I'll check it when I get home